.github/ISSUE_TEMPLATE/EDITOR_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "\U0001F50D Editor's Issue Template"
+about: Template to propose editing a chapter
+title:
+labels:
+assignees: ''
+
+---
+
+<!--
+Please complete the following sections when you open an issue. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. If you have write access to the repository please also assign the appropriate label (or labels) to your issue. Note that text within html comment tags will not be rendered.
+-->
+### Summary
+
+<!-- Please provide a detailed description of the chapter/subchapter you propose to edit, review or restructure. 
+Please provide as much context as possible and link to related issues and/or pull requests.
+-->
+
+*Lorem ipsum dolor sit amet, consectetur adipiscing.*
+
+### What needs to be done?
+
+<!-- In this section please comment on the overall structure of the file. Your response can cover the following questions: Do sections need clearer headings? Are there any technical terms that might not be understood? Is something missing that is worth discussing before adding?
+We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
+
+- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
+- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
+
+
+### Who can help?
+
+<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
```

Comment:
```suggestion
<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here.
If you know the original author the chapter/section, it may be useful to tag them here for their input.
-->
```